css/styles.css

```diff
@@ -18,77 +16,44 @@ header {
   color: white;
 }
 
-input {
-  margin-left: 1px;
-  margin-top: 7px;
-  width: 100%;
-}
-
-.button {
-  margin-left: 300px;
-}
-
-@media screen and (width > 0) {
-  p {
-    margin-bottom: 5px;
-  }
-
-  @media screen and (width > 424px) {
-    p {
-      margin-bottom: 6px;
-    }
-  }
-
-  @media screen and (width > 767px) {
-    p {
-      margin-bottom: 5px;
-    }
-  }
+.full-column {
+  flex-basis: 100%;
 }
 
 @media screen and (width > 567px) {
   .column-half {
     flex-basis: 50%;
+    margin-bottom: 15px;
   }
```

Comment:
Ideally your `utility` classes should only do one job, if it's `column-half` it should only change the width or flex basis. But you should already have `row`, `column-full`, and `column-half` in your `layout.css` file with correct media queries for them.